### ISSUE TITLE
Install script depends on jq and fails when not installed

### ISSUE THREAD

**Issue by shirts** [liamg/shox#2]
```
--- shox/scripts ‹master› » bash install.sh                                                                                                                                                                                                                                                                                                                           127 ↵
Determining platform...
Finding latest release...
install.sh: line 8: jq: command not found
(23) Failed writing body
```

**Comment by Saleh7** [comment]
Download jq

**Debian** and **Ubuntu** repositories. Install using `sudo apt-get install jq`

**Fedora** repository. Install using `sudo dnf install jq`

**openSUSE** repository. Install using `sudo zypper install jq`

**Arch** repository. Install using `sudo pacman -Sy jq`

## OR

https://stedolan.github.io/jq/download/

**Comment by liamg** [comment]
Fixed in #4
